packages/cli-test/src/cli/cli-process.ts

```diff
@@ -24,13 +31,15 @@ export interface SlackCLIGlobalOptions {
    * @description Whether the CLI should skip updating (`--skip-update`). Defaults to `true`.
    */
   skipUpdate?: boolean;
-  /**
-   * @description workspace or organization name or ID to scope command to
-   */
+  /** @description Workspace or organization name or ID to scope command to. */
```

Comment:
Could be more specific for the org case although `workspace or org ID` is also accurate; eg. `The ID of your team. if you are using a Standard Slack plan, this is your workspace ID. If you are using an Enterprise Grid plan, this is the organization ID, even if your app is only granted to a subset of workspaces within the org.`